src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx

```diff
@@ -1,9 +1,684 @@
+      width: 910,
+      height: 345,
+    }
+  )
+
+  return (
+    <GridColumns>
+      <Column span={6}>
+        <ResponsiveBox aspectWidth={4300} aspectHeight={1632} maxWidth="100%">
+          <Image
+            src={image.src}
+            width="100%"
+            height="100%"
+            srcSet={image.srcSet}
+            lazyLoad
+            alt=""
+          />
+        </ResponsiveBox>
+      </Column>
+      <Column span={6}>
+        <Text variant="sm" textTransform="uppercase" mb={1}>
+          Experience
+        </Text>
+        <Text variant="lg" mb={1}>
+          Trusted Marketplace for Both Commercial and Benefit Auctions
+        </Text>
+        <Text variant="sm">
+          Our roster of global auction house partners is rapidly expanding,
+          including, Bonhams, Phillips, Heritage, Rago/Wright, and Hindman
+          Auctions. To date, Artsy benefit auctions have helped to raise more
+          than $25 million for museums, nonprofit organizations, and other
+          charitable causes.
+        </Text>
+      </Column>
+    </GridColumns>
+  )
+}
+
+const PartnerWithArtsyForm: React.FC = () => {
+  useLoadScript({
+    id: "marketo-form",
+    src: "https://app-ab14.marketo.com/js/forms2/js/forms2.min.js",
+    onReady: () => {
+      // @ts-ignore
+      window.MktoForms2.loadForm(
+        "//app-ab14.marketo.com",
+        "609-FDY-207",
+        SHOULD_WARN_PRODUCTION_ENVIRONMENT
+          ? MARKETO_FORM_IDS.production
+          : MARKETO_FORM_IDS.development
+      )
+    },
+  })
+
+  return (
+    <GridColumns>
+      <Column span={6}>
+        <Text variant="lg" mb={1}>
+          Interested in Partnering with Artsy?
+        </Text>
+        <Text variant="sm">Apply to host your auctions on Artsy</Text>
+      </Column>
+      <Column span={6}>
+        <form id="mktoForm_4419" style={{ display: "none" }}></form>
+        <Formik
+          initialValues={{
+            Company: "",
+            FirstName: "",
+            LastName: "",
+            Title: "",
+            Website: "",
+            Email: "",
+            B2B_TOU_Active_Consent__c: false,
+            Type__c: "Gallery",
+          }}
+          onSubmit={values => {
+            // @ts-ignore
+            const form = window.MktoForms2.allForms()[0]
+            form.addHiddenFields(values)
+            form.submit()
+          }}
+          validationSchema={Yup.object().shape({
+            Company: Yup.string().required("Required"),
+            FirstName: Yup.string().required("Required"),
+            LastName: Yup.string().required("Required"),
+            Title: Yup.string(),
+            Website: Yup.string()
+              .url("Invalid website url")
+              .required("Required"),
+            Email: Yup.string()
+              .email("Invalid email address")
+              .required("Required"),
+            B2B_TOU_Active_Consent__c: Yup.boolean().oneOf([true], "Required"),
+            Type__c: Yup.string(),
```

Comment:
We reverse engineered the field names from inspecting the embedded Marketo form submission.